process_killer/src/main.cpp

```diff
@@ -3,6 +3,13 @@
 #include <windows.h>
 #include <ctime>
 
+/* Sleep between "min" and "max" seconds */
+void sleepRandomSeconds(int min, int max)
+{
+    const int sleep_time = (rand() % (min - max)*10) + min*10;
+    Sleep(sleep_time * 100);
```

Comment:
```suggestion
    sleep(sleep_time * 100);
```
Sleep, nie powinno być z małej litery?